react-client/src/Components/Helpfull.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+
+
+
+class Helpfull  extends React.Component{
+    constructor(props) {
+        super(props);
+        this.state ={
+            yes: props.yes,
+            no: props.no,
+            helpfullClick: props.helpfullClick,
+            id: props.id,
+            Report: false
+        }
+    }
+    clickHandler(param) {
+        console.log('hit')
+        if (param === 'yes') {
+            this.setState({
+                yes: this.state.yes + 1
+            })
+            this.state.helpfullClick('yes', this.state.id)
+        } else {
+            this.setState({
+                no: this.state.no + 1
+            })
+            this.state.helpfullClick('no', this.state.id)
+        }
+    }
+    Report() {
+        this.setState({
+            Report: true
+        })
+    }
+    render() {
+        if (!this.state.Report) {
```

Comment:
I'm interested in the if statement and how your system changes depending on the Boolean and what changes the Boolean.

Reply:
its only for the reported button so that it changes from a button to plain text like the original page